include/ptp_datacodes.h

```diff
@@ -158,6 +158,8 @@
 /* Android Random I/O Extensions Codes */
 #define PTP_OC_ANDROID_BEGINEDITOBJECT                  0x95C4
 #define PTP_OC_ANDROID_ENDEDITOBJECT                    0x95C5
+#define PTP_OC_ANDROID_GETPARTIALOBJECT                 0x95C1
```

Comment:
We strive for system independence. Eliminating all the Tizen-specific code was a considerable amount of work.

That said, it makes sense to support features that libmtp can excercise.

Reply:
Sorry for delay response. Yeah, i understand the principle to make it platform independent. However, i think, the events supported with this commit is actually independent of any platform & required to support file operations.

Purpose of this events :
Actually for file duplication & rename operations, MTP events of GETPARTIALOBJECT, BEGINEDITOBJECT, ENDEDITOBJECT in sequence is sent. And is in same alignment with libmtp working scenario.

In cmtp , get_object and set_object has the required logic for copy of files in different folders.
But before GET_OBJECT & SET_OBJECT is sent (in case of rename & duplication), BeginEdit & EndEdit event is triggered and expects to return success. Hence, in current BEGINEDIT, ENDEDIT returning OK will not break the MTP flow for the rename operations.

With these MTP event support, rename & duplication is successful. Please let me know if this is ok ?

Reply:
I tested your changes on a device. Rename seems to work ok. Thanks for taking care of this. Can you improve the commit messages?

Reply:
Thank you. Sure, will update with more descriptive commit message and other suggestions.

Reply:
Also, based on review comments and suggestions, i have updated and pushed as new PR. Can you please review that PR